docs/UserGuide.adoc

```diff
@@ -259,5 +342,17 @@ Example: `clone i/1`
 * <<ListTag, *List*>>: `list [m/MONTH] [t/TAG]` +
 Example: `list m/Jan t/Food`
 * <<HelpTag, *Help*>>: `help [COMMAND]` +
+* <<TagTag,*Tag*>>: `tag i/INDEX t/TAG_NAME` +
+Example: `tag i/8 t/Debt`
+* <<UntagTag, *Untag*>>: `untag i/INDEX t/TAG_NAME` +
+Example: `untag i/7 t/Important`
+* <<ConvertTag, *Convert*>>: `convert f/ORIGIN_CURRENCY t/TARGET_CURRENCY (v/VALUE|i/INDEX)` +
```

Comment:
Prefix should also be changed here.

Reply:
I have changed it to `c/TARGET_CURRENCY` only and removed ORIGIN_CURRENCY as it was actually redundant. This bottom part was updated as well.

Reply:
I have changed the command format to `convert to/TARGET_CURRENCY (c/COST | i/INDEX)`. This bottom part was updated as well

Reply:
Great! Thank you 😁